core/frontend/src/components/ping/ping1d.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <v-card style="min-width: 280px;">
+    <v-card-title class="justify-center pb-1">
+      <v-icon
+        x-large
+        class="ping1d"
+        color="primary"
```

Comment:
That will change if someone uses a custom color scheme, as distributors, we should enforce the same blue that we use.